### ISSUE TITLE
training image is the high resolution pictures?

### ISSUE THREAD

**Issue by scotthuang1989** [alexjc/neural-enhance#192]
I am new to this model. after reading the README. I tried the pre-built model to deblur my image. But the result is not good.  I want to train my own model with the image that are similar with my target image(I wan see the license plate):

Target:
![1](https://user-images.githubusercontent.com/5325686/34465344-f609d1e6-eee2-11e7-966d-a28932303f21.jpg)
 
So, what I should do is collect  images with cars in it and train it? Is there any requirements or pre-processing that I should apply to my images before feed them into model?

**Comment by rachitagrawal** [comment]
I am also looking for this answer? Will the training images be upscaled image or down-scaled image?

**Comment by rachitagrawal** [comment]
@scotthuang1989  Based on my analysis till now, it has to be HR (High-resolution) image. You can read Section 2 of "Photo-Realistic Single Image Super-Resolution Using a Generative Adversarial Network". This work is inspired from the above paper and they talk about the methodology in some detail.

**Comment by scotthuang1989** [comment]
@rachitagrawal  thanks. it does, I recently study the original paper, now I am running it, I downscale HR image as LR image.

**Comment by rachitagrawal** [comment]
@scotthuang1989 Original Paper?? You mean the SRGAN paper? Are you downscaling the images yourself? As far as I see the model does it own it's own (atleast in the training phase). I still don't understand though that why is --generator-downscale=0 and --generator-upscale=2. Any idea on that?

**Comment by scotthuang1989** [comment]
yes, I mean this [paper](https://arxiv.org/abs/1609.04802), in the section 3.2. they describe how they got the training data:

> We obtained the LR images
> by downsampling the HR images (BGR, C = 3) using
> bicubic kernel with downsampling factor r = 4.
